tests/testthat/test-test_dir.R

```diff
@@ -33,3 +33,27 @@ test_that('filter_test_scripts() with tricky names', {
     files[1]
   )
 })
+
+test_that("test_dir and `wrap`", {
+  ## forced to test screen output because tests outside of `testthat` blocks are
+  ## not recorded in return value, even when `wrap=TRUE` (see #596")
+
+  w.wrap <- capture.output(
+    test_dir("test_dir", filter = "bare-expectations", wrap = TRUE)
+  )
+  wo.wrap <- capture.output(
+    test_dir("test_dir", filter = "bare-expectations", wrap = FALSE)
+  )
+  # With wrap should display two dots to screen
+
+  expect_true(sum(grepl("^Bare: \\.\\.", crayon::strip_style(w.wrap))) == 1L)
+
+  ## Without wrap should not display any dots; however, it is not possible to
+  ## test this because this test itself is in a `test_that` block, and further
+  ## since the whole test suite is effectively run with `wrap=TRUE` it is not
+  ## possible to test this at all.  I ran this outside of the `test_that` call
+  ## and it worked as expected.
+
+  skip("untestable within test suite")
```

Comment:
Forgot to mention that we don't need the `skip()` if we're not planning to implement a test. The rest looks good to me.